doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -422,6 +426,17 @@ The EVP_PKEY_CTX_get0_ecdh_kdf_ukm() macro gets the user key material for B<ctx>
 The return value is the user key material length. The resulting pointer is owned
 by the library and should not be freed by the caller.
 
+=head2 ECDSA K (nonce) type parameters
+
+The EVP_PKEY_CTX_set_ecdsa_nonce_type() macro sets the ECDSA K (nonce) type to
```

Comment:
Change macro to method.

Reply:
@slontis Really? It is a marco, but not a method

Reply:
My mistake :)